nautobot/users/models.py

```diff
@@ -255,7 +255,7 @@ class ObjectPermission(BaseModel, ChangeLoggedModel):
     groups = models.ManyToManyField(to=Group, blank=True, related_name="object_permissions")
     users = models.ManyToManyField(to=settings.AUTH_USER_MODEL, blank=True, related_name="object_permissions")
     actions = JSONArrayField(
-        base_field=models.CharField(max_length=30),
+        base_field=models.CharField(max_length=255),
```

Comment:
May not need to change this one since it's an inner `base_field` for a JSONArrayField?